### PR TITLE
Add spatial object observer documentation

### PR DESCRIPTION
**Pull request by davidkline-ms**
This change adds a page describing the spatial object mesh observer and how it can be used in the editor to simulate spatial awareness using an imported 3D model.

PLEASE NOTE: this change is being targeted at the development branch to take advantage of the docs verification automation. Once approved and passing, it will be retargeted to the stabilization branch for merging.

### CODE REVIEW

**Comment by davidkline-ms** [comment]
doc checks passed. update base to stabilization branch

**Comment by wiwei** [comment]
I've retargetted this to look at mrtk_development instead of stabilization because that's where the docfx stuff gets generated out of

File: Documentation/SpatialAwareness/SpatialObjectMeshObserver.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+## Observer behavior notes
+
+Since the spatial object mesh observer loads data from a 3D model, it does not honor some of the standard mesh
+observer settings. The following sections describe these settings.
+
+### Update Interval
+
+The spatial object mesh observer sends all meshes to an application when the model is loaded. It does not
+simulate dime deltas between updates.
```
Comment (wiwei) [inline Documentation/SpatialAwareness/SpatialObjectMeshObserver.md]:
```suggestion
simulate time deltas between updates.
```

File: Documentation/SpatialAwareness/SpatialObjectMeshObserver.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+## Getting started
+
+To enable the spatial object mesh observer, you will need to configure the spatial awareness system.
+
+1. Add a new spatial observer
+
+![Add Spatatial Observer](../../Documentation/Images/SpatialAwareness/AddObserver.png)
+
+> {!NOTE]
```
Comment (wiwei) [inline Documentation/SpatialAwareness/SpatialObjectMeshObserver.md]: Nit: I would delete this note (it's proliferated in other areas as well, but honestly I think right here it doesn't add that much.

**Comment by wiwei** [comment]
Is there also going to be an example scene showing how this works (i.e. with some pre-canned object or scan)?

File: Documentation/SpatialAwareness/SpatialObjectMeshObserver.md (hunk 2 above)
Comment (davidkline-ms) [reply Documentation/SpatialAwareness/SpatialObjectMeshObserver.md]: sounds good

**Comment by davidkline-ms** [comment]
>Is there also going to be an example scene showing how this works (i.e. with some pre-canned object or scan)?

I could add one if you believe it is valuable. Turning on the observer will accomplish this in the editor with no extra work :)

**Comment by keveleigh** [comment]
>I've retargetted this to look at mrtk_development instead of stabilization because that's where the docfx stuff gets generated out of

If we end up adding docs for specific versions (or mrtk_release, etc), that'll cause this release to have out of date docs.

**Pull request merged**